packages/cli/src/commands/generate/auth/auth.js

```diff
@@ -114,7 +121,7 @@ export const description = 'Generate an auth configuration'
 export const builder = (yargs) => {
   yargs
     .positional('provider', {
-      choices: ['netlify', 'auth0', 'magic-link'],
+      choices: ['netlify', 'auth0', 'magic-link', 'firebase'],
```

Comment:
Could this be built dynamically from a directory list of `cli/src/commands/generate/auth/providers`? Then we'd never have to maintain it manually when new providers come online.

Reply:
I duplicated the SUPPORTED_PROVIDERS logic to use it for cli options. Same logic but I filtered out the README.md - which was appearing in SUPPORTED_PROVIDERS.
I would have kept my filtered version and only one constant for this, but maybe we want to keep both uses separated, what do you think?

Reply:
I think we can just keep that single SUPPORTED_PROVIDERS constant and use it here. Good catch filtering out README.md, I forgot about that one!